html/stile_VG.css

```diff
@@ -1,976 +1,490 @@
 @font-face
 {
-    font-family:zillaslab;
+    font-family: zillaslab;
     src: url("./zillaslab.otf");
 }
+
 @font-face
 {
-    font-family:zillaslab;
+    font-family: zillaslab;
     src: url("./zillaslab_i.otf");
-    font-style:italic;
+    font-style: italic;
 }
+
 @font-face
 {
-    font-family:zillaslab;
+    font-family: zillaslab;
     src: url("./zillaslab_b.otf");
-    font-weight:bold;
+    font-weight: bold;
+}
+
+:root
+{
+    --colore-testo-principale: black;
+    --colore-sfondo-principale: white;
+    --colore-testo-titoli: white;
+    --colore-sfondo-titoli: black;
+    --colore-testo-sceglifirefox: white;
+    --colore-sfondo-sceglifirefox: black;
+    --colore-testo-titolisezioni: white;
+    --colore-sfondo-titolisezioni: #222222;
 }
 
 #body
 {
-    font-family:zillaslab;
-    width:26.19cm;
-    height:37.1cm;
-    background-color:white;
-    border:0.04cm solid black;
-    position:absolute;
-    overflow:hidden;
-    font-size:0.5cm;
-    text-align:center;
-    top:0cm;
-    left:0cm;
-    right:0cm;
-    bottom:0cm;
+    font-family: zillaslab;
+    width: 26.19cm;
+    height: 37.1cm;
+    background-color: var(--colore-sfondo-principale);
+    border: 0.04cm solid black;
+    position: absolute;
+    overflow: hidden;
+    font-size: 0.5cm;
+    text-align: center;
+    top: 0cm;
+    left: 0cm;
+    right: 0cm;
+    bottom: 0cm;
 }
 
 div#titolo_principale
 {
-    position:absolute;
-    text-align:center;
-    top:0cm;
-    left:0cm;
-    right:0cm;
-    width:auto;
-    font-size:1.23cm;
-    font-family:inherit;
-    font-weight:bold;
-    background-color:black;
-    color:white;
-    padding-bottom:0.20cm;
-    height:1.48cm;
-    overflow:hidden;
+    position: absolute;
+    text-align: center;
+    top: 0cm;
```

Comment:
qui c;è una ripetizione presente in body.
Io farei tipo una classe di reset che imposta tutti questi valori predefiniti  ovunque così le classi successive sono più leggere

Reply:
Non ho capito. In che modo proponi? 🤔

Reply:
```
div {

Reply:
Ma il top cambia ad ogni elemento, così come anche il left e il right... l'unico che forse è abbastanza comune è l'overflow e la position... questo intendi tu?

Reply:
si cambia ma si usa il reset https://meyerweb.com/eric/tools/css/reset/ che mette dei valori predefinit così non vanno ridefiniti in tutti alleggerendo il codice

Reply:
Il "reset" sarebbe la "definizione" all'inizio dei valori più comuni e poi, nei singoli elementi "diversi" viene cambiato, giusto?

Reply:
si, si può dire anche così :-)

Reply:
Lo faccio subito, non ti avevo capito. Perdono xD